docs/team/limerencee.adoc

```diff
@@ -71,13 +70,16 @@ This makes the **delete** command syntax consistent with the **update** command.
 ** Updated the original **find** command to search for the specified keyword(s) in a transaction's `**Remark**` as well as `**Description**`. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/163[#163])
 
 * Documentation:
-** Updated the original developer guide to suit the context of `**THRIFT**`. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/169[#169])
+** Updated the architecture diagrams in the original developer guide to suit the context of `**THRIFT**`. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/169[#169])
+** Updated the user guide to acknowledge the original codebase.
 
 * Community:
-** Reviewed pull requests with (non-trivial comments): https://github.com/AY1920S1-CS2103T-W12-2/main/pull/69[#69], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/168[#168], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/174[#174], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/175[#175]
+** Reviewed pull requests of new features (with non-trivial comments): https://github.com/AY1920S1-CS2103T-W12-2/main/pull/69[#69], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/168[#168], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/174[#174], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/175[#175]
 
 == 3. Contributions to the User Guide
-The following sections illustrate my ability in writing documentation targeting end-users which guides them in using the various features of the application.
+The following section illustrates my ability in writing documentation targeting end-users which guides them in using the various features of THRIFT.
```

Comment:
This sentence seems pretty long, maybe you could consider breaking it up into 2 sentences or simplifying the sentence.